### DISCUSSION TITLE
ERR_OUT_OF_RANGE: data is too long

### DISCUSSION THREAD

**Discussion by alanmarkoTrilitech** [vercel/community#1823] [Help]
### Summary

Hey,

My relatively simple flask-based application runs quickly and correctly with just "vercel dev". However, I am getting the following errors after running "vercel" command:

`
Failed to process build result for "api/index". Data: {"type":"Lambda"}.
Error: data is too long
ERR_OUT_OF_RANGE: data is too long
`

With "vercel dev" it starts within two minutes, however the build takes more than 10 minutes until it gets to the errors with just vercel.

What causes this?

Thank you!

**Comment by amyegan** [comment]
Hi @alanmarkoTrilitech. Are you processing large files, like large image files or big data sets? It sounds like you may be running into the Serverless Function [payload size limit](https://vercel.com/docs/concepts/limits/overview#serverless-function-payload-size-limit). If that is the case, we have a [guide on strategies to work around the payload size limit](https://vercel.com/guides/how-to-bypass-vercel-body-size-limit-serverless-functions) that can help.

If that isn't the case, please check the [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#) for more error details and share any additional information you have. If you have a public repo or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that would also be helpful to share here if you still need debugging assistance
